docs/content/modules/products/admin/manage-products.mdx

```diff
@@ -389,10 +389,12 @@ If you don’t pass the `handle` body parameter, the handle of the product will
 
 ### Product Variant Prices
 
-It’s recommended to set the prices of product variants as the actual amount multiplied by a hundred. This behavior is implemented within the Medusa admin, and the Next.js storefront divides all prices by a hundred before displaying the price.
+For currencies that are not zero-decimal, you must set the prices of product variants as the actual amount multiplied by a hundred. This behavior is implemented within the Medusa admin, and the Next.js storefront divides all prices by a hundred before displaying the price.
```

Comment:
thought: Since this is the admin guide, I would actually omit this section. Think it can lead to more confusion than clarification. 

But keep the reference to the other docs explaining the concept.

Reply:
I shortened down the details but still kept the mention of how the pricing should be, since it's important from an admin perspective 👍🏻